packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -36,13 +41,24 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook service started on ${message.data.listen}. To open pipboard, please visit https://pipboard.vercel.app`);
+      let serverInfo;
+      // TODO(Feely): the listen field is deprecated, we will remove this
+      if (message.data.listen) {
+        serverInfo = message.data.listen;
```

Comment:
```suggestion
        serverInfo = `http://localhost:${message.data.listen}`;
```

Reply:
Because we use cli like:
```sh
$ picpook job list -h localhost -p 6927
```
so, i think `localhost:6927` is better than `http://locahost:6927`.

Reply:
User can directly just click the `http://locahost:6927` to open it in the browser.